controllers/resources/processor.go

```diff
@@ -4,14 +4,272 @@
 package resources
 
 import (
+	"context"
+	"fmt"
+
+	capsulev1beta2 "github.com/clastix/capsule/api/v1beta2"
```

Comment:
As the CI is complaining, please, sort imports.

Reply:
done